alchemy/src/sentry/client-key.ts

```diff
@@ -0,0 +1,294 @@
+import type { Context } from "../context.js";
+import { Resource } from "../resource.js";
+import { SentryApi, type SentryApiOptions } from "./api.js";
+
+/**
+ * Properties for creating or updating a ClientKey
+ */
+export interface ClientKeyProps {
+  /**
+   * The name of the key
+   */
+  name?: string;
+
+  /**
+   * Rate limit configuration
+   */
+  rateLimit?: {
+    /**
+     * Time window in seconds
+     */
+    window: number;
+
+    /**
+     * Maximum number of events allowed in the window
+     */
+    count: number;
+  };
+
+  /**
+   * The use case for the key
+   */
+  useCase?: "user" | "profiling" | "escalating_issues" | "tempest" | "demo";
+
+  /**
+   * The project slug that owns the key
+   */
+  project: string;
+
+  /**
+   * The organization ID or slug that owns the key
+   */
+  organization: string;
+
+  /**
+   * Auth token to use (overrides environment variable)
+   */
+  authToken?: SentryApiOptions["authToken"];
```

Comment:
This should be `Secret` instead of `string`. Also better not to reference `SentryApiOptions["authToken"]`. `Secret` is both more concise and obvious.

Reply:
Will do. This was leftover from when I did `authToken?: 'sntry_u_${string}'` syntax.

Reply:
Resolved via 274939a5c3123e1003bcdb36d106ad9eaca431af